### DISCUSSION TITLE
Anything to be done with the 25% of files with unknown dates?

### DISCUSSION THREAD

**Discussion by BrandonSchaaf** [TheLastGimbus/GooglePhotosTakeoutHelper#255] [I NEED HELP]
I'm joining many others in thanking you for this tool!

Of my 11,684 photos and videos, 2,921 went into the date-unknown folder. Those files, of course, do have dates in Google Photos. Is there anything to be done about these? Maybe this is a job for the exiftool, but I looked into the tool and can't understand how exactly it could help.

**Comment by TheLastGimbus** [comment]
> Is there anything to be done about these?

There is :tada: - show their filenames/folder structure/their jsons etc here, and we may try to understand why gpth didn't find them

If you are sure some of them do have exif metadata (for example, you can see their date in some photo app), you could send them (the photos itself, not just their names) and i could try to find out why gpth didn't read it

**Comment by IZA165** [comment]
I've got the same problem, around 16,000 images out of my 53,000 have unknown dates, but I have the json files for these. How would I batch merge the jsons with the unknown images?

**Reply by TheLastGimbus** [reply]
> How would I batch merge the jsons with the unknown images?

if i would know answear to that question, i would built that into gpth, right? Please read the comment i made above

**Reply by IZA165** [reply]
Thanks, sorry

**Reply by IZA165** [reply]
I think it's because the photos have the have the same name or because they have ().

**Reply by TheLastGimbus** [reply]
Investigate that further, try to find what cases it doesn't work, and show us the screenshots etc here after that :tada:

**Reply by IZA165** [reply]
Trying out the latest release to see if that makes a difference

**Reply by TheLastGimbus** [reply]
No it doesn't. You made this issue in november, and latest update was september

Maybe actually do what i said above? Find which photos it doesn't work for, what are their names, are there corresponding jsons for them, etc - and show the screenshots here

**Reply by IZA165** [reply]
![image](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/144132079/1aa41308-993e-4d0e-9f5f-8fc6b7f654d8)
not sure why these files with normal names don't have any date

**Reply by TheLastGimbus** [reply]
Looks like this may be because everything is uppercase. Do some investigations that I said about above
